index.html

```diff
@@ -6,15 +6,179 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>ANVIO</title>
   <link rel="stylesheet" href="css/reset.css">
+
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick-theme.css"/>
+  
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/media.css">
 
+  
   <script src="js/main.js"></script>

```

Comment:
just interested why this not with other scripts at the bottom?

Reply:
Header is fixed.

I added 2 script files at the bottom according to the slick slider library documentation. I added the first file at the top because  using the event 'load'.